components/pirateweather.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "PirateWeather",
   "owner": ["@alexander0042"],
-  "manifest": "https://raw.githubusercontent.com/alexander0042/pirate-weather-hacs/master/custom_components/darksky/manifest.json",
-  "url": "https://github.com/alexander0042/pirate-weather-hacs"
+  "manifest": "https://raw.githubusercontent.com/alexander0042/pirate-weather-ha/master/custom_components/PirateWeather/manifest.json",
```

Comment:
This file does not exist.
The folder is still named "darksky"
If you are changing the folder name, I suggest "pirateweather" instead of "PirateWeather"